ibtk/src/lagrangian/LDataManager.cpp

```diff
@@ -1288,6 +1288,12 @@ LDataManager::mapPETScToLagrangian(std::vector<int>& inds, const int level_numbe
     return;
 } // mapPETScToLagrangian
 
+AO&
+LDataManager::getAO(int level_number)
+{
+    return d_ao[level_number];
```

Comment:
Could you add an index check here (via `TBOX_ASSERT()`)?

Reply:
Done